harmonica/filters/_filters.py

```diff
@@ -60,5 +60,478 @@ def derivative_upward_kernel(fft_grid, order=1):
+    # Convert frequencies to wavenumbers
+    k_northing = 2 * np.pi * freq_northing
+    # Compute the filter for northing derivative in frequency domain
+    da_filter = (k_northing * 1j) ** order
+    return da_filter
+
+
+def upward_continuation_kernel(fft_grid, height):
+    r"""
+    Filter for upward continuation in frequency domain
+
+    Return a :class:`xarray.DataArray` with the values of the frequency domain
+    filter for computing the upward continuation. The filter is built upon the
+    frequency coordinates of the passed ``fft_grid`` and is defined as follows:
+
+    .. math::
+
+        g(\mathbf{k}) = e^{|\mathbf{k}| ^ h}
+
+    where :math:`\mathbf{k}` is the wavenumber vector
+    (:math:`\mathbf{k} = 2\pi \mathbf{f}` where :math:`\mathbf{f}` is the
+    frequency vector) and :math:`h` is the height of the upward continuation.
+
+    Parameters
+    ----------
+    fft_grid : :class:`xarray.DataArray`
+        Array with the Fourier transform of the original grid.
+        Its dimensions should be in the following order:
+        *freq_northing*, *freq_easting*.
+        Use :func:`xrft.xrft.fft` and :func:`xrft.xrft.ifft` functions to
+        compute the Fourier Transform and its inverse, respectively.
+    height : float
+        The height of upward continuation. Value should be negative. It has the
+        same units as the input xarray data.
+
+    Returns
+    -------
+    da_filter : :class:`xarray.DataArray`
+        Array with the kernel for the upward continuation filter in frequency
+        domain.
+
+    References
+    ----------
+    [Blakely1995]_
+
+    See also
+    --------
+    harmonica.upward_continuation
+    """
+    # Catch the dims of the Fourier transformed grid
+    dims = fft_grid.dims
+    # Grab the coordinates of the Fourier transformed grid
+    freq_easting = fft_grid.coords[dims[1]]
+    freq_northing = fft_grid.coords[dims[0]]
+    # Convert frequencies to wavenumbers
+    k_easting = 2 * np.pi * freq_easting
+    k_northing = 2 * np.pi * freq_northing
+    # Compute the filter for upward continuation in frequency domain
+    da_filter = np.exp(np.sqrt(k_easting ** 2 + k_northing ** 2) * height)
+    return da_filter
+
+
+def gaussian_lowpass_kernel(fft_grid, wavelength):
+    r"""
+    Filter for gaussian low-pass in frequency domain
+
+    Return a :class:`xarray.DataArray` with the values of the frequency domain
+    filter for computing the northing derivative. The filter is built upon the
+    frequency coordinates of the passed ``fft_grid`` and is defined as follows:
+
+    .. math::
+
+        g(\mathbf{k}) = e^\frac{-|\mathbf{k}| ^ 2}{\mathbf{kw}^2}
+
+    where :math:`\mathbf{k}` is the wavenumber vector
+    (:math:`\mathbf{k} = 2\pi \mathbf{f}` where :math:`\mathbf{f}` is the
+    frequency vector) and `\mathbf{kw}` is the wavenumber of the cutoff
+    wavelength
+    :math:`\mathbf{w}` (:math:`\mathbf{kw} = \frac{2\pi} {\mathbf{w}}`).
```

Comment:
I think the cutoff wavenumber should be a scalar. And I think we can improve its notation:
```suggestion
        g(\mathbf{k}) = e^\frac{-|\mathbf{k}| ^ 2}{k_\text{cutoff}^2}

    where :math:`\mathbf{k}` is the wavenumber vector
    (:math:`\mathbf{k} = 2\pi \mathbf{f}` where :math:`\mathbf{f}` is the
    frequency vector) and :math:`k_\text{cutoff}` is the cutoff wavenumber:
    :math:`k_\text{cutoff} = \frac{2\pi}{\lambda_\text{cutoff}}`, 
    where :math:`\lambda_\text{cutoff}` is the cutoff wavelength.
```